docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Sequences and Strings
+menuPosition: 3
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run readme:update" from the root of the repo. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_ROOT:path=packages/dds/sequence/README.md&start=2) -->
+Every item in a SharedSegmentSequence is at a specific position starting at 0, kind of like an array. However, it
+differs from an array in that the positions can move as local and remote collaborators make modifications to the
+sequence. There are a number of different sequence types:
+
+- SharedString for storing and collaborating on a sequence of text.
+- SharedNumberSequence for storing and collaborating on a sequence of numbers.
+- SharedObjectSequence for storing and collaborating on a sequence of JSON-serializable objects.
+
+As the name suggests SharedSegmentSequence, or sequence for short, are made of segments. Segments are the leaf nodes of
+the tree data structure that enables collaboration and backs the sequence. Segments may be split and merged as
+modifications are made to the sequence. Every segment has a length from 1, to the length of the sequence. The length of
+the sequence will be the combined length of all the segments.
+
+When talking about positions in a sequence we use the terms *near*, and *far*. The nearest position in a sequence is 0,
+and the farthest position is its length. When comparing two positions the nearer position is closer to 0, and the
+farther position is closer to the length.
+
+## Using a Sequence
+
+Sequences support three basic operations: insert, remove, and annotate.
```

Comment:
There was no mention to properties, so "annotate" here may feel unfamiliar when it is introduced here.